lib/rmt/file_validator.rb

```diff
@@ -0,0 +1,59 @@
+require 'fileutils'
+require 'rmt/checksum_verifier'
+
+class RMT::FileValidator
+  class << self
+    def validate_local_file(file_reference, deep_verify:)
```

Comment:
We're using this `deep_verify` variable, but I can only find one usage, and it's non-variable.

```
    return false if ::RMT::FileValidator.validate_local_file(file, deep_verify: false)
```

Does it need to be a variable at all?

Reply:
In fact, I let this `deep_verify` variable to implement the `--deep-verify` flag for the `rmt-cli mirror` command, in the other PR we did checksum verifying for all cases, and @ikapelyukhin pointed it could make the command slower. I didn't add the flag into this PR to not make it bigger yet. I can remove and readd it once we add the flag.

Reply:
@lcaparroz I'd suggest actually finishing the feature in one PR.
This PR is one PR too many already, because I can't easily see the full diff (without merging both PRs into a separate branch and diffing locally).

Reply:
I'll do a follow up PR after we merge this one and #585. Adding the flag together with these changes will make it even harder for review.